integration/networking/port_mapping_linux_test.go

```diff
@@ -178,3 +204,96 @@ func TestProxy4To6(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Check(t, is.Equal(resp.StatusCode, 404))
 }
+
+// TestAccessPublishedPortFromHost checks whether published ports are
+// accessible from the host.
+func TestAccessPublishedPortFromHost(t *testing.T) {
+	// Both IPv6 test cases are currently failing in rootless mode. This needs further investigation.
+	skip.If(t, testEnv.IsRootless)
+
+	ctx := setupTest(t)
+
+	assert.NilError(t, exec.Command("sysctl", "net.ipv6.conf.all.disable_ipv6=0").Run())
+	out, err := exec.Command("ip", "addr", "add", "fdfb:5cbb:29bf::2/64", "dev", "eth0", "nodad").CombinedOutput()
```

Comment:
Is this something that needs to be reverted after the test? Can this cause issues when running these tests on the host (not in a container)?